tests/test_types.py

```diff
@@ -31,4 +41,63 @@ def test_count(count, truth):
 def test_dict():
     now = dt.datetime.utcnow()
     model = QueryParams(count=1, created_before=now)
-    assert model.dict() == dict(count=1, created_before=now.isoformat())
+    assert model.dict() == dict(count=1, created_before=now.isoformat() + 'Z')
+
+
+def test_json_encoder():
```

Comment:
You're effectively writing two tests. Could generalize and parameterize the test?